### DISCUSSION TITLE
An option to disable blur on scroll for articles with featured images?

### DISCUSSION THREAD

**Discussion by loiSvelasco** [nunocoracao/blowfish#222] [Q&A] [answered]
I was trying to find a way to disable blur on scroll when the hero style for a featured image is set to background reading an article, is this possible? I admit I may not have looked hard enough if this is possible but reading through the docs I am unable to find it. I guess the blur is needed if the background has conflicting colors which makes the text hard to read and such which is understandable. Thank you!

**Comment by nunocoracao** [comment, marked as answer]
Great ask @loiSvelasco, I've implemented it for the homepage but not for the article view... it's a simple addition I will take a look at it.

**Reply by loiSvelasco** [reply]
That was quick!! Thank you! I was thinking of forking and creating a PR but then I realized I dunno how hugo works, or how go works, or how this is built so yeah, thanks again!

**Reply by nunocoracao** [reply]
If you want to have a go I always appreciate help :)

**Comment by nunocoracao** [comment]
@loiSvelasco look at https://github.com/nunocoracao/blowfish/issues/230

**Reply by loiSvelasco** [reply]
thank you!
